lib/nx_zlib.h

```diff
@@ -113,11 +113,12 @@ struct selector {
 #define HEADER_ZLIB  1
 #define HEADER_GZIP  2
 
+/* 47 is the max windowBits for auto detect header */
 #ifndef MAX_WBITS
-#  define MAX_WBITS 15
+#  define MAX_WBITS 47
 #endif
 #ifndef DEF_WBITS
-#  define DEF_WBITS MAX_WBITS
+#  define DEF_WBITS 47
```

Comment:
Is this change necessary? The new value of `MAX_WBITS` would have set `47` here as well.

Reply:
Seems zlib is defining `MAX_WBITS` somewhere. Without this change, at least in the servers I tested, `DEF_WBITS` is always 15 and libnxz assumes we have a zlib header.